secondary_bulk_insert_script.sql

```diff
@@ -117,3 +117,47 @@ VALUES
 		(7, NULL, 'Groote Schuur drive', CAST(-33.939424 AS Decimal(10, 6)), CAST(18.464692 AS Decimal(10, 6))),
 		(4, 6, 'Guild road', CAST(-26.180933 AS Decimal(10, 6)), CAST(28.018597 AS Decimal(10, 6)))
 GO
+
+INSERT INTO [dbo].[Incident]
+		([User_ID]
+		,[Location_ID]
+		,[Date]
+		,[Time]
+		,[Description])
+	VALUES 
+		(9, 1, '03/05/2021', '10:11', 'Money was stolen out of my pocket while walking down the street.'),
+		(5, 7, '19/06/2021', '2:02', 'I was pulled into a car, drugged, driven to an unknown location and left to die.'),
+		(8, 9, '06/05/2021', '1:05', 'I was punched from behind and the attacker ran away for some unknown reason.'),
+		(1, 7, '15/08/2021', '15:19', 'I was told to go back to my own country, because I am ruining this country.'),
+		(6, 6, '17/02/2022', '17:00', 'I was told that my kind are not human.'),
+		(15, 10, '23/12/2021', '11:33', 'I was spiked and then raped.'),
+		(12, 12, '29/03/2021', '18:02', 'While walking back from class myself and a group of friends were held at gun point and forced to give up our cellphones.'),
+		(13, 19, '05/12/2021', '9:59', 'My friend was working from back work, when a man in a hoodie murdered him.'),
```

Comment:
Line 135:
Is it suppose to say "My friend was walking back from work ... "?

Reply:
I was trying to make one for reporting the murder. There were also a few others where people report seeing abuse as well. But if you guys would prefer only reporting incidents that happen to you I can change them quick?

Reply:
I just don't understand what  "My friend was working from back work"

Reply:
I was just giving more context to the description but I can remove it if you would like?

Reply:
Nah, I think this is just me maybe :')

Reply:
Okay well we can get everyone else's opinion as well. I really don't mind changing it if you feel it is unnecessary.